docs/input/docs/reference/configuration.md

```diff
@@ -552,6 +552,11 @@ Example of invalid `Strict`, but valid `Loose`
 1.2.3.4
 ```
 
+### handle-detached-branch
```

Comment:
Since you're targeting `main`, this is going to be rolled into v6. To target v5, you need to rebase and retarget against `support/5.x`, but I would like to avoid the introduction of new configuration properties. As such, I don't think we need any new configuration properties to add support for detached HEAD?